libs/ui/pagination/helm/src/lib/hlm-pagination-link.directive.ts

```diff
@@ -47,6 +47,7 @@ export class HlmPaginationLinkDirective {
 	protected readonly _computedClass = computed(() =>
 		hlm(
 			paginationLinkVariants(),
+			this.link() === undefined ? 'cursor-pointer' : '',
```

Comment:
@ajitzero I apply the `cursor-pointer` in the pagination link. This also applies the style to pagination next/previous as they use the pagination link directive as well. 

The `routerLink` type can be `string | any[] | UrlTree | null | undefined`, maybe its sufficient to only apply the styles when no link is applied === `undefined`. What do you think?

Reply:
Yes, that makes sense! Thanks!